admin/data/sql/migrations/20200915160026.do.create-gias-staging-table.sql

```diff
@@ -0,0 +1,15 @@
+
+/**
+  Temporary for gias exploration. will be removed before PR finalised
+**/
+CREATE TABLE [mtc_admin].[giasStaging](
+	[id] [int] IDENTITY(1,1) NOT NULL PRIMARY KEY,
+	[createdAt] [datetimeoffset](3) NOT NULL DEFAULT GETUTCDATE(),
+	[updatedAt] [datetimeoffset](3) NOT NULL DEFAULT GETUTCDATE(),
+	[version] [timestamp] NOT NULL,
+	[leaCode] [int] NULL,
```

Comment:
Not sure why leaCode and estabCode may be nullable.  Is it because they are determined from the dfeNumber in later processing?

Reply:
completeness of establishment entries in the incoming extract were unknown, so this is just to tolerate errors.

its a temporary fixture for easy querying of extract data that will be removed before PR is finalised